tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+
+### !callout-info
+
+## Confidently Make Hypotheses
+
+As Leila works through this, they will need to make a lot of assumptions. Leila can take the time to fact-check all of her assumptions, which is a completely valid thing to do (and typically a good strategy for life). However, in this situation, Leila can _also_ confidently rely on their skills, intuition, context, and help from others to make theories about what's going on.
+
+<br/>
+
+"Maybe the `test` folder only holds test files, and we have a `.test.js` file for each function." Making theories like this will help Leila move forward in their learning. It will also help them correct themself and their working knowledge more easily!
+
+### !end-callout
+
+### Hypothesizing About `describe`
+
+Let's look at block of code that begins on the first line, and closes on the last line.
+
+<!-- prettier-ignore-start -->
+```js
+describe('fizzBuzz', () => {
+  // ...
+});
+```
+<!-- prettier-ignore-end -->
+
+Even though we don't know what `describe` means here, we can see that its syntax looks similar to a function call with two arguments:
+
+<!-- prettier-ignore-start -->
+```js
+describe( ..., ...
+);
+```
+<!-- prettier-ignore-end -->
+
+We can also see that the first argument is the string literal `'fizzBuzz'`. When Leila sees this, Leila questions, "What is this string literal used for?"
+
+When scanning through the rest of this code, we don't see this string literal being used anywhere obviously, so they will pause on that question for now.
+
+The second argument seems to be a large, anonymous arrow function with zero parameters, that contains even more unfamiliar syntax.
+
+<!-- prettier-ignore-start -->
+```js
+() => {
+    // ...
+}
+```
+<!-- prettier-ignore-end -->
+
+Let's approach the contents of this function piece-by-piece.
+
+### Hypothesizing About `test`
+
+Inside the anonymous function started in `describe`, there is a block of code that begins with `test()`.
+
+<!-- prettier-ignore-start -->
+```js
+  test('returns Fizz on multiples of 3', () => {
+    // ...
+  });
+```
+<!-- prettier-ignore-end -->
+
+This is also a function being invoked. This `test` function seems to take in two arguments, the string literal `'returns Fizz on multiples of 3'` and an anonymous arrow function.
+
+Leila notices that this pattern feels very similar to the `describe` block. 🤔 They should keep that observation, because it's a good one!
+
+### Arrange
+
+Inside the next anonymous arrow function, we see the following code:
+
+<!-- prettier-ignore-start -->
+```js
+    // Arrange
+    const num = 9;
+```
+<!-- prettier-ignore-end -->
+
+It's a comment indicating an "Arrange" step and a `const` variable `num` assigned to the number `9`.
+
+Leila is an overthinker, and they could dig deeper into what this line means. However, this time, they will choose to make only one theory: the variable `num` will be used later in the test.
```

Comment:
rephrase to avoid using overthinking in what sounds like a pejorative sense
```suggestion
Leila usually likes to deeply investigate any new concepts they encounter, and they could dig deeper into what this line means. However, this time, they decide to move on with the following hypothesis: the variable `num` will be used later in the test.
```